libraries/llvm/array.c

```diff
@@ -0,0 +1,45 @@
+#ifndef EFFEKT_ARRAY_C
+#define EFFEKT_ARRAY_C
+
+void c_array_erase_fields(void *envPtr) {
```

Comment:
Could you add one comment above that explains in a picture / sentence how arrays are represented?